docs/source/release-notes.rst

```diff
@@ -52,14 +52,14 @@ App
 - Fixed batch selection with ctrl + click in the grid
   `#5046 <https://github.com/voxel51/fiftyone/pull/5046>`_
 
-FiftyOne Teams 2.1.2
+FiftyOne Enterprise 2.1.2
 --------------------
 *Released October 31, 2024*
 
 - Fixed an issue that prevented `delegation_target` from being properly set when
   running delegated operations with orchestrator registration enabled
 
-FiftyOne Teams 2.1.1
+FiftyOne Enterprise 2.1.1
```

Comment:
Let me know if there's a preference for the release notes that reference previous versions of the tool to remain "Fiftyone Teams" since that is what they were called at the time. I don't know if this would help with searching or to minimize confusion.